src/style/Common.scss

```
@@ -0,0 +1,60 @@
+@import url('https://fonts.googleapis.com/css2?family=Lobster&family=Lobster+Two:ital@0;1&family=Roboto&display=swap');
+
+:root {
+  /* color */
+  --color-light-grey: #dbdbdb;
+  --color-grey: #c7c7c7;
+  --color-dark-grey: #747272;
```

```suggestion
  $color-light-grey: #dbdbdb;
  $color-grey: #c7c7c7;
  $color-dark-grey: #747272;
```
변수를 잘 만들어서 사용하셨는데, 우리는 지금 scss를 사용하고 있느니 이에 맞는 방법으로 변수를 선언해서 사용하시면 더 좋습니다!
위 코드와 같은 방법으로 변수를 선언하고, 사용하고자 하는 각각의 scss에서 import해서 사용하시면 됩니다!